src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/github-connector.css

```diff
@@ -0,0 +1,79 @@
+.create-github-api-key-container {
```

Comment:
Include a *namespace* to avoid problems in the future. My recommendation:

```
#github-branch-source .create-github-api-key-container {
  [...]
}
```

I use the `artifactId` as namespace and everything related to this plugin (UI obviously):

```
<div id="github-branch-source">
  [...]
</div>
```

It is only semantic.